internal/exec/stages/mount/mount.go

```diff
@@ -133,6 +135,9 @@ func (s stage) mountFs(fs types.Filesystem) error {
 	}
 
 	if _, err := os.Stat(path); err != nil && os.IsNotExist(err) {
+		if _, err := util.PathExists(path); err != nil {
+			pathExists = true
```

Comment:
We don't need to introduce a variable for this, and it makes the code harder to follow.  Just move the relabeling call directly after the directory creation in the "doesn't exist" path.

And actually, `firstMissing` is only used during the directory creation, so its calculation can be moved into the "doesn't exist" path as well.

Reply:
makes sense have that changed now!